must-gather/collection-scripts/gather_crs

```diff
@@ -24,33 +24,17 @@ echo "Starting collection of: [${resources[@]}]"
 
 # we use nested loops to nicely output objects partitioned per namespace, kind
 for resource in ${resources[@]}; do
-  echo "Collecting ${resource}"
-  /usr/bin/oc get ${resource} --all-namespaces -o custom-columns=NAME:.metadata.name,NAMESPACE:.metadata.namespace --no-headers 2> /dev/null | \
-  while read ocresource; do
-    ocobject=$(echo $ocresource | awk '{print $1}')
-    ocproject=$(echo $ocresource | awk '{print $2}')
-    if [ -z "${ocproject}" ]|[ "${ocproject}" == "<none>" ]; then
-      object_collection_path=/must-gather/clusters/${clusterID}/cluster-scoped-resources/${resource}
-      mkdir -p ${object_collection_path}
-      /usr/bin/oc get ${resource} -o yaml ${ocobject} &> ${object_collection_path}/${ocobject}.yaml &
+  /usr/bin/oc get ${resource} --all-namespaces -o custom-columns=NAMESPACE:.metadata.namespace --no-headers 2> /dev/null | \
```

Comment:
I think you could also add here `| unique`, because multiple resources may be in one namespace. Then per namespace get the resource(s).